src/notation/view/noteinputbarcustomisemodel.cpp

```diff
@@ -73,8 +73,6 @@ QVariant NoteInputBarCustomiseModel::data(const QModelIndex& index, int role) co
     case ItemRole: return QVariant::fromValue(item);
     default: return SelectableItemListModel::data(index, role);
     }
-
-    return QVariant();
```

Comment:
I believe some compilers (gcc?) would warn that the function does not always have a return value (which is nonsense). The solution to make them all happy would be:
```cpp
    default: break;
    }

    return SelectableItemListModel::data(index, role);
}
```

Reply:
Here MSVC complained about statement without effect

Reply:
But your proposal works too, so let's use that.

Reply:
I see, but that would also be fixed with my proposed solution, without causing a (bogus) warning on Gcc.